src/Spryker/Client/RabbitMq/Model/Consumer/Consumer.php

```diff
@@ -58,7 +58,7 @@ public function receiveMessages($queueName, $chunkSize = 100, array $options = [
         $rabbitMqOption = $options['rabbitmq'];
 
         $this->channel->basic_qos(null, $chunkSize, null);
-        $this->channel->basic_consume(
+        $consumerTag = $this->channel->basic_consume(
```

Comment:
Could you please describe why do we need this change for?

Reply:
Consumer has to be canceled after we received all messages we need, otherwise it's just hangs and ends only when process ends. Also consumer runs as a "service" and takes new messages from the queue as soon as we acknowledge one of already processed. And we don't need this behaviour, this also causes problems when running this method multiple times during one process.

This has to be reflected in release notes, I think :)